stepper-lib/src/execution/air/mod.rs

```diff
@@ -80,19 +75,19 @@ macro_rules! log_instruction {
         );
 
         log::debug!(
-            target: crate::log_targets::CALL_EVIDENCE_PATH,
-            "  current call evidence path: {:?}",
-            $call_ctx.current_path
+            target: crate::log_targets::EXECUTED_TRACE,
```

Comment:
Should it be EXECUTION_TRACE or EXECUTED_TRACE? What's the intention:
- trace of what? => of execution
- what kind of trace? => executed trace (and there might be an execut**ING** trace or something)